src/main/java/kitchenpos/table/domain/TableGroup.java

```diff
@@ -0,0 +1,78 @@
+package kitchenpos.table.domain;
+
+import static kitchenpos.table.domain.OrderTables.MIN_ORDERTABLES_SIZE;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.util.CollectionUtils;
+
+@Entity
+public class TableGroup {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @CreatedDate
+    private LocalDateTime createdDate;
+    @Embedded
+    private OrderTables orderTables = new OrderTables();
+
+    public TableGroup() {
```

Comment:
```suggestion
    protected TableGroup() {
```

잘 작성해주셨는데 빠진 것 같아서 체크했어요 :)

Reply:
앗 감사합니다 😭